fixtures-service/json_helpers.py

```diff
@@ -0,0 +1,15 @@
+import json
+import os.path
+
+
+def load(filename):
+    with open(filename) as file:
+        data = json.load(file)
+    return data
+
+
+def dump(obj, fullpath, sort_keys=True):
```

Comment:
The `sort_keys` options is added here because sometimes it's not a good idea to have the fields sorted. For example the whole `Shopozor.json` must not have its fields sorted alphabetically, as the field order matters for insertion into the database.